source/server/server.cc

```diff
@@ -246,6 +246,14 @@ void InstanceImpl::flushStatsInternal() {
 
 bool InstanceImpl::healthCheckFailed() { return !live_.load(); }
 
+ProcessContextOptRef InstanceImpl::processContext() {
+  if (process_context_ == nullptr) {
+    return absl::nullopt;
+  } else {
```

Comment:
nit: No need for the else since the if block returns.

Reply:
Done.